payments-core/src/main/java/com/stripe/android/cards/DefaultCardAccountRangeStore.kt

```diff
@@ -44,7 +44,8 @@ internal class DefaultCardAccountRangeStore(
     internal fun createPrefKey(bin: Bin): String = "$PREF_KEY_ACCOUNT_RANGES:$bin"
 
     private companion object {
-        private const val PREF_FILE = "InMemoryCardAccountRangeSource.Store"
+        private const val VERSION = 2
+        private const val PREF_FILE = "InMemoryCardAccountRangeSource.Store.$VERSION"
```

Comment:
Updating the key here so that we can ”force-refresh”. I’m doing this because we might already be in a corrupted state for existing users.